libdevcrypto/CryptoPP.cpp

```diff
@@ -27,7 +27,7 @@
 #include <libdevcore/Assertions.h>
 #include <libdevcore/SHA3.h>
 
-static_assert(CRYPTOPP_VERSION == 570, "Wrong Crypto++ version");
+static_assert(CRYPTOPP_VERSION == 565, "Wrong Crypto++ version");
```

Comment:
Do we downgrade?

Reply:
Yes. 5.6.5 is the last released version. Previously we used 5.6.5 + my commit fixing some compiler warnings. But we don't have care about warnings because Hunter is hiding them.